index.coffee

```diff
@@ -24,7 +24,7 @@ writeFiles = (bundles, sprockets, tmpPath) ->
   writtenFiles = []
 
   for bundle in bundles
-    asset = sprockets.findAsset bundle
+    asset = sprockets.findAsset(if bundle.pattern? then bundle.pattern else bundle)
```

Comment:
Lets construct a default config object here and not have to worry about whether properties are available or not later. e.g.

```
_ = require "underscore"

if typeof bundle == "string"
  fileConfig = 
    pattern: bundle
else
  fileConfig = bundle

_.defaults fileConfig:     
  included: true
  served: true
  watched: true
  nocache: false

```

You can then get rid of the `?= true`